### PR TITLE
Solves #109, extended User-Agent data

### PR DESCRIPTION
**Pull request by nerdguru**
The new code I added to extend the User-Agent data passes the linter, but an existing line now gets barked at for reasons that aren't clear, so I thought I'd submit to see if someone more expert could figure that out or if it'll go away.

The only part I was unsure of was the ability to set the two optional fields using environment variables.  I went the quick route and pull the variables inside restsession.py but if you want me to follow the model of environments.py I can do that and resubmit.  Since the paramaters are optional and the API key set by environments.py is required, I wasn't sure which way would be preferable.

### CODE REVIEW

**Comment by cmlccie** [comment]
Hi Pete (@nerdguru), thank you for submitting this.  I would like to play with this header (see what the finished results look like) and pull the header generation code out of the `__init__()` method.  I think we could put it in a `user_agent()` function or method *similar* to what pip does.

**Comment by cmlccie** [comment]
I will push this to a feature branch and work on the code a bit.  *-Thank you!*

**Comment by nerdguru** [comment]
Sounds good to me, thanks!

File: webexteamssdk/restsession.py [hunk 1]
```diff
@@ -48,6 +52,7 @@
     check_response_code, check_type, extract_and_parse_json, validate_base_url,
 )
 
+from webexteamssdk._version import get_versions
 
 # Helper Functions
```
Comment (sQu4rks) [inline webexteamssdk/restsession.py:57]: flake8 on the CI is failing because there is a blank line missing here.

**Comment by cmlccie** [comment]
This functionality has been merged in #123.  Thank you @nerdguru!

**Pull request closed without merging**